espanol/banquete/banquete.md

```diff
@@ -0,0 +1,3 @@
+[El cocinero fue a controlar si tenia suficiente comida. Abrio la despensa y se dio cuenta que..](..comida/comida.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.